src/codecs/html/README.md

```diff
@@ -38,3 +38,126 @@ Some of the main external dependencies:
 - `js-beautify` for pretty generated HTML.
 
 - `json5` for more human readable representation of arbitrary values.
+
+---
+
+## Microdata and semantic tag usage (WIP)
+
+### Schema nodes to add microdata to
+* Article
+    * `div` `itemtype='schema:Article'`
+    * itemprops:
+        * authors* (author?): `ol`
+        * title*: `h1`? (+ `title`)
+        * environment: `data`?
+* Audio
+    * `audio`
+    * itemprops:
+        * caption*: `figcaption`
+        * transcript
+* Brand
+    * `picture`/`img` `itemtype='schema:Brand'`
+    * itemprops:
+        * name: maybe unnecessary, just use in alt tag?
+        * logo*: `source`/`image`
+        * reviews
+* Code
+    * `div` `itemtype='stencila:Code'`
+    * itemprops:
+        * language* (maybe important for setting syntax highlighting): `data`
+        * value*: `code`
+* CodeChunk
+    * `div` `itemtype='stencila:CodeChunk'`
+    * itemprops:
+        * outputs: `ol` of `samp`/`output` (possibly include `figure`/`table`)
+* CodeExpression
+    * `div` `itemtype='stencila:CodeExpr'`
+    * itemprops:
+        * value:`code` (syntax highlighting from Code node's language prop)
+* Collection
+    * `nav` `itemtype='schema:Collection'`
+    * itemprops:
+        * parts: `ol` of CreativeWorks/Articles (just title, author, and URIs/relative links?)
+* ContactPoint
+    * `address` `itemtype='schema:contactPoint'`
+    * itemprops:
+        * availableLanguages: `ul` of `data`? if not `p` (maybe unnecessary for styling)
+        * emails: `ol` of `a href`
+        * telephone: `a href`
+* CreativeWork:
+    * `div` `itemtype='schema:CreativeWork'`
+    * itemprops:
+        * authors: `ol` of `span` Person/Organization itemtypes.
+        * citations: `ol` of title/authors/`a href`
+        * content: `article`/`div` with `sections`
+        * dateCreated: `span time`
+        * dateModified: `span time`
+        * datePublished/date: `span time`
+        * editors: `ol` of `span` Person itemtypes
+        * funders: `ol` of `span` Person/Organization itemtypes
+        * isPartOf: `data`?
+        * licenses: `ol` of `a href`
+        * parts/hasPart: `ol` of figure/table/object/URI (maybe unnecessary for styling)
+        * publisher: `span` Person/Organization itemtype
+        * text: `data`? (maybe unnecessary for styling)
+        * title/headline: `h1`
+        * version: `span`
```

Comment:
This isn't related to this specific schema, but thinking of including all these properties in the output seems a bit overwhelming to me. Not sure how we handle going about laying all this information out in a useful way without being simply cluttered and noisy 😬

Reply:
Just use CSS to make it look good 🤷‍♂️